app/views/mods_panel.py

```diff
@@ -1799,6 +1801,13 @@ def recalculate_internal_errors_warnings(self) -> None:
         logger.info(f"Finished recalculating {self.list_type} list errors")
         return total_error_text, total_warning_text, num_errors, num_warnings
 
+    def _has_replacement(self, dep: str, package_ids_set: Set[str]) -> bool:
```

Comment:
I'd use the Python3.9+ built in set instead of the typing Set so then eventually can just get rid of the typing import

Reply:
Done, thanks!